scripts/99-image_check.sh

```diff
@@ -542,6 +542,8 @@ elif [[ $OS == "CentOS Stream" ]]; then
         ost=1
     if [[ $VER == "8" ]]; then
         osv=1
+    elif [[ $VER == "9" ]]; then
```

Comment:
Debian 12 was released recently, should we add support for that too?

Reply:
This is not us, this script is provided by DO https://github.com/digitalocean/marketplace-partners/blob/master/scripts/99-img-check.sh